.github/workflows/release.yml

```diff
@@ -26,11 +31,79 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  # build-release-windows: # this is done by hand because of signing
   # build-release-macos: # LF volunteer
 
+  build-release-win:
+    runs-on: windows-latest
+    if: ${{ true }}  # change to false to skip if release is built by hand
```

Comment:
I don't really  see  us making commits just to flip this, in all likelihood  we'd  just delete the unsighed  build and replace it with a signed one.